README.md

```diff
@@ -25,6 +25,8 @@ For more information on how to contribute to Refuge Restrooms, or how the techno
 
 If you just want to get your environment set up for making changes locally and testing, you can head directly to [CONTRIBUTING.md](https://github.com/RefugeRestrooms/refugerestrooms/blob/develop/CONTRIBUTING.md).
 
+See also our [Code of Conduct](https://github.com/RefugeRestrooms/refugerestrooms/blob/develop/CODE_OF_CONDUCT.md) for our standards of community, and who is responsible for enforcement, and where to direct complaints.
```

Comment:
"enforcement" --> sounds too harsh? Or maybe it's just authoritative, in a way that is appropriate for framing a code of conduct.

Reply:
I might just c/p from above to here `Please also read our [Code of Conduct](https://github.com/RefugeRestrooms/refugerestrooms/blob/develop/CODE_OF_CONDUCT.md), which gives guidance on our standards of community and interaction.`